etc/samples/logging.yaml.journald.sample

```diff
@@ -0,0 +1,37 @@
+version: 1
+disable_existing_loggers: False
+formatters:
+    simple:
+        format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
```

Comment:
Please remove "name" and add below items

"[%(filename)s:%(lineno)s - %(funcName)20s() ] %(message)s"